packages/neutron-sdk/src/stargate/aux.rs

```diff
@@ -0,0 +1,28 @@
+use cosmwasm_std::{Binary, CosmosMsg, Deps, QueryRequest, StdResult};
+use prost_types::Timestamp as TimestampGen;
+use serde::de::DeserializeOwned;
+
+pub(crate) fn make_stargate_query<Req, Res>(deps: Deps, req: Req, path: &str) -> StdResult<Res>
+where
+    Req: prost::Message,
+    Res: DeserializeOwned,
+{
+    deps.querier.query(&QueryRequest::Stargate {
+        path: path.to_string(),
+        data: req.encode_to_vec().into(),
+    })
+}
+
+pub(crate) fn create_stargate_msg<Req: prost::Message>(req: Req, path: &str) -> CosmosMsg {
```

Comment:
I'd add a doc comment for the func, that describes a purpose of the function, arguments and a returned value